### DISCUSSION TITLE
Lightning not syncing

### DISCUSSION THREAD

**Discussion by BeatGroove** [raspiblitz/raspiblitz#4815] [Q&A]
I'm having issues with my node. I haven't really touched it for months, and everything was running fine until one day Lightning stopped working. Now it just says "Lightning progress : prepare sync". It has been like that for about a week now.

I reflashed the sd card a couple of times, both with v1.10 and 1.11.2 but that didn't help.

You can see the debug log here: http://termbin.com/2mjm

I see there are some core lightning errors that say **BROKEN** gossipd: backtrace: ... but I don't know what that means.

Blitzapi logs also says "CLN ConnectionRefusedError. Retrying in 10 seconds."

Does anyone have an idea how I might fix this? Any help is greatly appreciated.

Thank you.

**Comment by rootzoll** [comment]
http://termbin.com/2mjm is already deleted. Have you tried the freshly released: RaspiBlitz v1.11.3 version?

**Comment by Godrigos** [comment]
That happened to me a while ago. Version 1.11.3 is syncing again and today reached 100%. Now, once I used SCB + SEED it is in recoveryscan processes. I don't know how long it takes but the syncing processe returned to normal.

**Comment by Godrigos** [comment]
I am getting the same behavior now on 1.11.4. It just started out of thin air. Blitz working fine and then got back to sync screen and never got out. Restart, reboot. Nothing changes. Quite wierd.
